inc/saiport.h

```diff
@@ -1698,6 +1814,68 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SYSTEM_PORT,
 
+    /**
+     * @brief To enable/disable low latency variation
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_LOW_LATENCY_VARIATION,
+
+    /**
+     * @brief Query the Auto Negotiated Resolved FEC
+     *
+     * @type sai_s32_list_t sai_port_fec_mode_t
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_AUTO_NEG_FEC_MODE,
```

Comment:
The existing SAI_PORT_ATTR_FEC_MODE attribute can be used to query the operational FEC mode. When AN is enabled, SAI_PORT_ATTR_ADVERTISED_FEC_MODE is to be used for the configured list of advertised FEC modes and SAI_PORT_ATTR_FEC_MODE can be queried to get the operational FEC mode. When AN is disabled, SAI_PORT_ATTR_FEC_MODE reflects the configured as well as operational FEC mode. 
Please capture your use-case if it cannot be fit in this sequence.

Reply:
SAI_PORT_ATTR_FEC_MODE attribute is used to configure port FEC.  SAI_PORT_ATTR_ADVERTISED_FEC_MODE  is used to set FEC when AN is enabled. SAI_PORT_ATTR_AUTO_NEG_FEC_MODE is the resolved AN FEC after AN completion.  Resolved/advertised/configured FEC values can be different.

Reply:
I suggest to separate this change from this PR. Is it independent from SyncE/low latency mode?